impl/java/core/src/main/kotlin/br/com/guiabolso/events/json/JsonAdapterExtensions.kt

```diff
@@ -0,0 +1,20 @@
+package br.com.guiabolso.events.json
+
+import kotlin.reflect.jvm.javaType
+import kotlin.reflect.typeOf
+
+inline fun <reified T> JsonAdapter.fromJsonOrNull(json: String): T? {
+    return runCatching { fromJson<T>(json, typeOf<T>().javaType) }.getOrNull()
```

Comment:
Usando o `runCatching` não pode dificultar o _troubleshooting_ caso o json esteja errado?

Reply:
O objetivo dessa função é devolver `null ` em casos onde não é possível criar a instância do objeto desejado, então acho que se esse não for o objetivo por qualquer motivo, o usuário deveria usar as funções `fromJson(...)`